### PR TITLE
Issue 532 should be done

### PR DESCRIPTION
**Pull request by vjcao**
views.py, .env, view_file_access_within_week.html, file_access_within_week.js, and settings.py were modified to function with lecture capture events and other file types in the future. Added .env and sql files as references of what was used to test the file view access portion of myLA.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Required to set this with some value
```
Comment (pushyamig) [inline .env]: `.env` should never is committed

**Comment by pushyamig** [comment]
The work done as part of the FE needs to be changed based on the React flow. The BE work is still valid. This PR need to adapt to React branch to get merged

**Comment by jonespm** [comment]
I have a incomplete PR that's changing the backend. I'll talk to @vjcao about it and he can incorporate that as well as rework this for React.

**Comment by vjcao** [comment]
Redoing #532 in react

**Pull request closed without merging**